src/util/get-config-files.ts

```diff
@@ -45,26 +47,25 @@ export async function getConfigFiles(
       path,
     });
 
-    return [requestedRepoFile, defaultRepoConfig];
```

Comment:
This used to return `[null, defaultConfig]`

So no reason to maintain the array on line 35: `const files = [requestedRepoFile];`

Reply:
Please see my other comment, I don't think it's a good idea to just not return `requestedRepoFile` anymore.